gemfile.d/test.rb

```diff
@@ -1,3 +1,4 @@
 group :test do
   gem "vcr", "~> 6.1"
+  gem "faraday-excon"
```

Comment:
Is this not pulled in by the new dependency for the client gems?

Reply:
The client gems don't require it, which is the problem. Faraday itself used to require faraday-excon, but no longer.

Reply:
It's a very recent change that broke some other plugins like foreman_azure_rm too.